static/app/views/projectInstall/createProject.tsx

```diff
@@ -474,16 +467,19 @@ export function CreateProject() {
 
       updateFormData('projectName', newName);
     },
-    [
-      updateFormData,
-      formData.projectName,
-      formData.platform?.key,
-      formData.platform?.category,
-      organization,
-    ]
+    [updateFormData, formData.projectName, formData.platform?.key, organization]
   );
 
-  const category: Category = formData.platform?.category ?? 'popular';
+  const platform = formData.platform?.key;
+  const defaultCategory = platform
```

Comment:
we need to clean up these types with keys too... everything should be PlatformIntegration (at the moment is very annoying to map everywhere, but there are so many interconnect places where updates are required)